app/screens/post_options/post_options.tsx

```diff
@@ -119,15 +131,15 @@ const PostOptions = ({
         );
     };
 
-    const additionalSnapPoints = 2;
+    const additionalSnapPoints = 1;
 
     return (
         <BottomSheet
             renderContent={renderContent}
             closeButtonId={POST_OPTIONS_BUTTON}
             componentId={Screens.POST_OPTIONS}
             initialSnapIndex={0}
-            snapPoints={[((snapPoints + additionalSnapPoints) * ITEM_HEIGHT), 10]}
+            snapPoints={[(((snapPoints + additionalSnapPoints) * ITEM_HEIGHT) + (canAddReaction ? REACTION_PICKER_HEIGHT : 0)), 10]}
```

Comment:
I think this condition should also be added to the 'gekidou' branch ?

```
     (canAddReaction ? REACTION_PICKER_HEIGHT : 0)), 10]}
```

Reply:
This is "solved" in a different way in gekidou, but I think it is broken in several places (fixed here).
Since the plan is to have eventually this branch into gekidou, I think we can wait until then :smile: